keyboards/luddite/keymaps/default/keymap.c

```diff
@@ -0,0 +1,36 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _BASE 0
+#define _FN1 1
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+};
+
+#define _______ KC_TRNS
```

Comment:
Lines 16 and 17 can be safely deleted. QMK defines these automatically.

https://github.com/qmk/qmk_firmware/blob/3875d6f5815bf263cb3ee5d18b74c5ec8ab865f2/quantum/quantum_keycodes.h#L29-L31

Reply:
Thanks, done!